test/integration/targets/vyos_interfaces/vars/main.yaml

```diff
@@ -0,0 +1,233 @@
+    - "set interfaces ethernet eth1 description 'Replaced by Ansible'"
+    - "set interfaces ethernet eth1 vif 100"
+    - "set interfaces ethernet eth1 vif 100 description 'VIF 100 - Replaced by Ansible'"
+    - "delete interfaces ethernet eth2 speed"
+    - "delete interfaces ethernet eth2 duplex"
+    - "delete interfaces ethernet eth2 vif 200 description"
+    - "set interfaces ethernet eth2 description 'Replaced by Ansible'"
+    - "set interfaces ethernet eth2 mtu '1400'"
+
+  after:
+    - name: "lo"
+      enable: true
+
+    - name: "eth1"
+      description: "Replaced by Ansible"
+      enable: true
+      vifs:
+        - vlan_id: 100
+          enable: true
+          description: "VIF 100 - Replaced by Ansible"
+
+        - vlan_id: 200
+          enable: true
+
+    - name: "eth2"
+      mtu: 1400
+      description: "Replaced by Ansible"
+      enable: true
+      vifs:
+        - vlan_id: 200
+          enable: true
+
+    - name: "eth0"
+      enable: true
+      duplex: "auto"
+      speed: "auto"
+
+overridden:
+  commands:
+    - "delete interfaces ethernet eth1 description"
+    - "delete interfaces ethernet eth1 speed"
+    - "delete interfaces ethernet eth1 duplex"
+    - "delete interfaces ethernet eth1 mtu"
+    - "delete interfaces ethernet eth1 vif 200 description"
+    - "delete interfaces ethernet eth2 speed"
+    - "delete interfaces ethernet eth2 duplex"
+    - "delete interfaces ethernet eth2 vif 200 description"
+    - "set interfaces ethernet eth2 description 'Overridden by Ansible'"
+    - "set interfaces ethernet eth2 mtu '1200'"
+
+  after:
+    - name: "lo"
+      enable: true
+
+    - name: "eth0"
+      enable: true
+      speed: "auto"
+      duplex: "auto"
+
+    - name: "eth1"
+      enable: true
+      vifs:
+        - vlan_id: 200
+          enable: true
+
+    - name: "eth2"
+      enable: true
+      description: "Overridden by Ansible"
+      mtu: 1200
+      vifs:
+        - vlan_id: 200
+          enable: true
+
+deleted:
+  commands:
+    - "delete interfaces ethernet eth1 description"
+    - "delete interfaces ethernet eth1 speed"
+    - "delete interfaces ethernet eth1 duplex"
+    - "delete interfaces ethernet eth1 mtu"
+    - "delete interfaces ethernet eth1 vif 200 description"
+    - "delete interfaces ethernet eth2 description"
+    - "delete interfaces ethernet eth2 speed"
+    - "delete interfaces ethernet eth2 duplex"
+    - "delete interfaces ethernet eth2 mtu"
+    - "delete interfaces ethernet eth2 vif 200 description"
+
+  after:
+    - name: "lo"
+      enable: true
+
+    - name: "eth0"
+      enable: true
+      speed: "auto"
+      duplex: "auto"
+
+    - name: "eth1"
+      enable: true
+      vifs:
+        - vlan_id: 200
+          enable: true
+
+    - name: "eth2"
+      enable: true
+      vifs:
+        - vlan_id: 200
+          enable: true
+
+round_trip:
+  after:
+    - name: "lo"
+      enable: true
+
+    - name: "eth0"
```

Comment:
here

Reply:
This is not a test. This is the fixtures file.